README.md

```diff
@@ -72,5 +96,8 @@ You can now upload questions_bb.txt to Blackboard as a test or question pool
 using the "Upload questions" button. The latex embedded with double dollar
 e.g. `$$y = x^2$$` in questions_bb.txt will not automatically render when
 uploaded as rendering latex is a feature of the web editor. This means that
-after uploading you have to edit each question and then save without chaging
-anything to triger hte latex rendering (maybe there is a better way...).
+after uploading you have to edit each question and then save without changing
+anything to trigger the latex rendering (maybe there is a better way...).
+[This help
+page](https://blackboard.secure.force.com/publickbarticleview?id=kA339000000L6QH)
+from blackboard seems to suggest there is not another way.
```

Comment:
There is the possibility of solving this by rendering offline if we can support the other Blackboard format though. Probably worth mentioning here.